tests/Feature/NewsTest.php

```diff
@@ -0,0 +1,130 @@
+
+        $this->post("/dashboard/news", [
+            "title" => "Example question",
+            "slug" => "example-question",
+            "content" => "Content of the wiki",
+        ])->assertSessionHasNoErrors();
+
+        $this->assertDatabaseCount("news", 1);
+    }
+
+    public function testSlugForNewsCanBeProvided(): void
+    {
+        $this->assertDatabaseCount("news", 0);
+
+        $this->post("/dashboard/news", [
+            "title" => "Example question",
+            "slug" => "example-question",
+            "content" => "Content of the wiki",
+        ])->assertSessionHasNoErrors();
+
+        $this->assertDatabaseCount("news", 1);
+
+        $this->assertDatabaseHas("news", [
+            "title" => "Example question",
+            "slug" => "example-question",
+            "content" => "Content of the wiki",
+        ]);
+    }
+
+    public function testSlugForNewsCanBeGenerated(): void
+    {
+        $this->assertDatabaseCount("news", 0);
+
+        $this->post("/dashboard/news", [
+            "title" => "Example question",
+            "content" => "Content of the wiki",
+        ])->assertSessionHasNoErrors();
+
+        $this->assertDatabaseCount("news", 1);
+
+        $this->assertDatabaseHas("news", [
+            "title" => "Example question",
+            "slug" => "example-question",
+            "content" => "Content of the wiki",
+        ]);
+    }
+
+    public function testNewsCanBeUpdated(): void
+    {
+        $news = News::factory()->create();
+
+        $this->assertDatabaseMissing("news", [
+            "title" => "Example question",
+            "slug" => "example-question",
+            "content" => "Content of the wiki",
+        ]);
+
+        $this->patch("/dashboard/news/{$news->id}", [
+            "title" => "Example question new",
+            "slug" => "example-question-new",
+            "content" => "Content of the wiki",
+        ])->assertSessionHasNoErrors();
+
+        $this->assertDatabaseHas("news", [
+            "title" => "Example question new",
+            "slug" => "example-question-new",
+            "content" => "Content of the wiki",
+        ]);
+    }
+
+    public function testNewsCannotBeCreatedWithInvalidData(): void
+    {
+        $this->post("/dashboard/news", [
+            "title" => Str::random(256),
+        ])->assertSessionHasErrors([
+            "title",
+        ]);
+
+        $this->assertDatabaseCount("news", 0);
+    }
+
+    public function testNewsCannotBeCreatedWithoutData(): void
+    {
+        $this->post("/dashboard/news", [
+        ])->assertSessionHasErrors([
+            "title",
+            "content",
+        ]);
+
+        $this->assertDatabaseCount("news", 0);
+    }
+
+    public function testNewsCanBeDeleted(): void
+    {
+        $news = News::factory()->create();
+
+        $this->assertDatabaseCount("news", 1);
+
+        $this->delete("/dashboard/news/$news->id");
+
+        $this->assertDatabaseCount("news", 0);
```

Comment:
Are u using `{ }` or not? One test above has `{ }`. 

```suggestion
        $this->delete("/dashboard/news/{$news->id}");

        $this->assertDatabaseCount("news", 0);
```

Reply:
Fixed i'm sticking with the version with `{}`